gravitee-am-service/src/main/java/io/gravitee/am/service/impl/ReporterServiceImpl.java

```diff
@@ -347,7 +346,7 @@ public String createReporterConfig(String domain) {
             // dash are forbidden in table name, replace them in domainName by underscore
             String tableSuffix = null;
             if (domain != null) {
-                tableSuffix = domain.replaceAll("-", "_");
+                tableSuffix = domain.replace("-", "_");
```

Comment:
As per the comment 
 ```
   // dash are forbidden in table name, replace them in domainName by underscore
```

Why use `replace` instead of `replaceAll` ?

Reply:
replaceAll uses regex as input

replace uses CharSequence as input

Reply:
Thanks. You reminded me how it was confusing the naming of these methods were 😄 , `replace` does indeed replace all.